lib/formatter-webpack.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const StylelintPluginError = require('./StylelintPluginError');
+
+module.exports = function webpack (items) {
+  const result = [];
+
+  for (const item of items) {
```

Comment:
What is your opinion on doing this with a `.reduce` and `.map`s? I understand this imperative version is probably simpler to read, and for now is all we need, but I like the "pure" versions because they are easier to test the individual units should they need to be extracted at some point. I took the liberty of writing this all out, just for my own understanding:

```js
module.exports = function webpackFormatter (items) {
  return items.reduce(function (result, item) {
    const file = item.source;

    const warnings = item.warnings.map(function (warning) {
      const line = warning.line;
      const column = warning.column;
      const text = warning.text;
      return new StylelintPluginError(file + '\n' + line + ':' + column + ' stylelint: ' + text);
    });

    const invalidOptionWarnings = item.invalidOptionWarnings.map(function (warning) {
      return new StylelintPluginError('stylelint\n0:0 ' + warning.text);
    });

    return result.concat(warnings, invalidOptionWarnings);
  }, []);
};
```

When the codebase is eventually upgraded to use things like destructuring and arrow functions this method will become much shorter :)